crypto/engine/eng_cnf.c

```diff
@@ -14,9 +14,9 @@
 
 /* ENGINE config module */
 
-static char *skip_dot(char *name)
+static const char *skip_dot(const char *name)
 {
-    char *p;
+    const char *p;
```

Comment:
merge in the assignment, add a blank line.  use p != NULL below.  and perhaps the ?: operator.